spyderlib/spyder.py

```diff
@@ -2735,6 +2754,11 @@ def restart(self):
         env['SPYDER_ARGS'] = spyder_args
         env['SPYDER_PID'] = str(pid)
         env['SPYDER_IS_BOOTSTRAP'] = str(is_bootstrap)
+        env['SPYDER_RESET'] = str(reset)
+        if os.name == 'nt':
+            env['PYTHONPATH'] = ';'.join(sys.path)
+        else:
+            env['PYTHONPATH'] = ':'.join(sys.path)

```

Comment:
This could cause issues with our consoles. Specially with our Mac app because it does some funky things to PYTHONPATH.

Question: I think we need this only for `bootstrap` to work, right @goanpeca?